utils/generateRandomNumber.c

```diff
@@ -0,0 +1,12 @@
+#include <stdio.h>
+#include "libs/mtwister/definitions/mtwister.h"
+
+int randNum()
+{
+  MTRand r = seedRand(1337);
+  int i;
```

Comment:
hi my name  is AAHHHH

Reply:
fixed at [7513ba6](https://github.com/jowjow22/simplified-c-blockchain/pull/1/commits/7513ba65563b425df211e982a9ce2f788d049b31)